_posts/2020-10-01-omero-5-6-3.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.3
+---
+
+Today we are releasing OMERO.server 5.6.3 which uses OMERO.blitz 5.5.8
+to expand the OMERO API's `Duplicate` functionality to allow the
+duplication of images, attachments, and other model objects that are
+backed by files on disk. Where possible, duplicated files are linked
+with originals to save space on underlying storage. This long-awaited
+ability to create a true copy of microscopy data can be invoked via the
+OMERO.cli duplicate plugin, available from PyPI; if you already use
+[`omero-cli-duplicate`](https://pypi.org/project/omero-cli-duplicate/)
+then we suggest upgrading to at least 0.4.0.
+
+When <a
+href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#manage-images-in-datasets-projects">managing
```

Comment:
I slightly worry about the use of "latest" here along with upcoming renamings, but I assume the linkchecker will detect all of that.